lib/media.js

```diff
@@ -199,7 +209,7 @@ function getStyleFiles(state) {
         );
         return acc;
       }, []);
-  // in edit mode, grab all variations
+    // in edit mode, grab all variations
```

Comment:
```suggestion
  // in edit mode, grab all variations
```

Reply:
Took me a long minute to figure out what the hell was going on here. 😬